### ISSUE TITLE
Collaborate with Engagement on CSV Senders upgrading their capabilities to HL7

### ISSUE THREAD

**Issue by dylansmith-Nava** [CDCgov/prime-reportstream#12604]
### User Story
As a ReportStream scrum team lead, I want to collaborate on possible engagement strategies with Engagement team leads regarding Senders' HL7 capabilities, to see if we can encourage CSV Senders to start sending in HL7.

### Background
Owing to the [CSV Support in the UP MVP](https://docs.google.com/document/d/1dfvSWQjPhAKHW2shQ2jn4VxAPrLpvNJ4duK27NY0gnU/edit#heading=h.1pruvfgadxev) Product Brief, some 50% of current COVID pipeline Senders send messages in CSV format. As we transition these Senders to the UP, we are considering various strategies to both 'meet users where they are' while encouraging more modern practices that will have more longevity going forward. Based on our knowledge at the moment, we have reason to suspect CSV Senders may be capable of sending in HL7 now.
- Since the beginning of the pandemic, it is likely that some Senders have invested in upgrading their technical abilities, and now can send via HL7
- If these Senders have connected to our API, there is a potential likelihood that they have the technical capacity to upgrade their message sending to HL7
- Since the beginning of the pandemic, some Senders have taken on the ability to report more conditions, and it would be valuable to send via HL7

### Risks/Impacts/Considerations
- Senders who currently send via CSV do not have the technical capabilities to upgrade their messaging format to HL7
- CSV Senders do not see the business value in upgrading to HL7 to send messages to ReportStream, especially if they are smaller senders
- CSV Senders attempt to upgrade to HL7 and run into issues, causing delays in the upgrade process -- or, after time and money spent, CSV Senders find that this upgrade process is more trouble than it is worth
- In the next pandemic, ReportStream would always be fielding CSV messages from Senders, so asking current Senders to migrate to HL7 is potentially only a temporary fix
- Other potential strategies, such as [in this ticket](https://app.zenhub.com/workspaces/platform-6182b02547c1130010f459db/issues/gh/cdcgov/prime-reportstream/12605), would take less time and technical capacity to implement

### Open Questions
- Which Senders should be engaged with first vs later, if this strategy is approved? Would we need to consider a long-term plan to expect a roll-out of these upgrades over a few quarters?
- How long should we expect a Sender to take to upgrade to sending an HL7 format?
- How should we support Senders in this process?
- How involved should ReportStream be in making sure that these Senders are successful -- and most importantly, do not fail at the upgrade?
- If this is our primary strategy, what should be our contingency plan if Senders don't want to or can't upgrade their processes?

### Links
- [Product Brief - CSV Support in the UP MVP](https://docs.google.com/document/d/1dfvSWQjPhAKHW2shQ2jn4VxAPrLpvNJ4duK27NY0gnU/edit#heading=h.atxy1smv1e88)
- [CSV>FHIR Research] (https://docs.google.com/document/d/1mk06z-gZa9YAJqYtuZ6aQ3hW3iWFFzsVKNfgYboH7wQ/edit)
- [CSV in UP MVP Requirements](https://docs.google.com/spreadsheets/d/1Mxtp-CYDaZxcSGVyZRiTGUVD93wPboBTT1Ik8uQ23CM/edit#gid=149930974)
- [COVID 19 CSV Schema](https://github.com/CDCgov/prime-reportstream/blob/master/prime-router/metadata/schemas/covid-19.schema)
- [CSV->FHIR Formatting & Gap Analysis #12043](https://app.zenhub.com/workspaces/platform-6182b02547c1130010f459db/issues/gh/cdcgov/prime-reportstream/12043)
- [CSV File Formats](https://docs.google.com/spreadsheets/d/1WumZYfUURgQWu1ihvK4NgUbR2ms0VE2MfzspDx22eTw/edit#gid=0)

### Acceptance Criteria
- [x] @Andrey-Glazkv to discuss this strategy with @LaFreda1 about the potential strategy to ask Senders about upgrading to HL7 capabilities
- [x] Note and synthesize pros and cons of this strategy in collaboration with Engagement
- [x] Note and synthesize the potential effectiveness of this strategy in relation to the other strategies being discussed

### Out of Scope
- Go-ahead/launching this strategy

**Comment by dylansmith-Nava** [comment]
Met with LaFreda today and discussed proactively what Engagement might need to do if this strategy is approved. Waiting on word from Product leads as to the direction of CSV strategy. We planned to meet up async next week, or on a live call if needed. CC @jillian-hammer

**Comment by dylansmith-Nava** [comment]
Meeting between LaFreda, Arnej, Andrey, Jess, and I today. The outcome of the meeting is that 1) Engagement will take over any and all next tickets of the work, and 2) I can conclude this ticket and my direct involvement. 

Action items coming out of the meeting (that can likely be translated into Engagement tickets, to be written by @LaFreda1 )
- Collaborate with @audreykwr on an appropriate communication template between ReportStream and Senders
- Generate a list of CSV Senders that ReportStream should reach out to
- Ensure that our preferred channel of communication is a) email via b) Customer Support
- Discuss cadence (likely, sending all the emails at once)

There will likely need to be new tickets on how to manage responses and how ReportStream might support an organization in their migration. @LaFreda1 

If this seems all agreeable, I will close out of this ticket here.

**Comment by dylansmith-Nava** [comment]
Based on the conversation in Slack with @LaFreda1 today, I can conclude this ticket as the Engagement ticket concepts have been ported over to the Engagement board. Thank you! @Andrey-Glazkv CC @jillian-hammer .
